06-hot-reloading-with-directives/services/reviews/schema.js

```diff
@@ -0,0 +1,68 @@
+const { makeExecutableSchema } = require('@graphql-tools/schema');
+const { stitchingDirectives } = require('@graphql-tools/stitching-directives');
+
+const NotFoundError = require('../../lib/not_found_error');
+const readFileSync = require('../../lib/read_file_sync');
+
+const { stitchingDirectivesTypeDefs, stitchingDirectivesValidator } = stitchingDirectives();
+
+const typeDefs = `
+  ${stitchingDirectivesTypeDefs}
+  ${readFileSync(__dirname, 'schema.graphql')}
+`;
+
+// data fixtures
+const usernames = [
+  { id: '1', username: '@ada' },
+  { id: '2', username: '@complete' },
+];
+
+const reviews = [
+  {
+    id: '1',
+    authorId: '1',
+    product: { upc: '1' },
```

Comment:
For simplicity in flat database modeling, I'd suggest making this a flat `productUpc: '1'` foreign key, then resolve it into an object like author is handled below...

```
product: (review) => ({ upc: review.productUpc }),
```

Reply:
I think this also is a straight copy from their demo